main.go

```diff
@@ -66,7 +65,6 @@ var (
 		"horizontalpodautoscalers": struct{}{},
 	}
 	availableCollectors = map[string]func(registry prometheus.Registerer, kubeClient clientset.Interface, namespace string){
-		"componentstatuses":        collectors.RegisterComponentStatusCollector,
```

Comment:
`componentstatuses` options need to be removed from `--collectors`

Reply:
@zouyee The default value for `--collectors` is of `defaultCollectors` keys which has `componentstatuses` removed.